### PR TITLE
Re-organize the consuming documentation for clarity

### PR DESCRIPTION
**Pull request by jeremycline**
A number of users did not find the section on using a class as a
callback. This attempts to re-organize the documentation around the
consumer API to make it clearer.

@dustymabe, @bowlofeggs, it'd be great if you could skim this and let me know if it's clearer.

### CODE REVIEW

**Comment by dustymabe** [comment]
I think it would have helped me today if we add an example of calling that example class via the CLI. i.e. `fedora-messaging --conf /work/my_config.toml consume --callback=modname:SaveMessage`

File: docs/consuming.rst [hunk 1]
```diff
@@ -79,39 +79,84 @@ Consult the manual page for complete details on this command line interface.
...
+The introduction contains a very minimal callback. This section covers the
+complete API for consumers.
 
-    $ systemctl start fm-consumer@sample.service  # uses /etc/fedora-messaging/sample.toml
+The Callback
+------------
+
+The callback provided to :func:`fedora_messaging.api.consume` or the command-line
+interface can be any callable Python object, so long as it accepts the message
+object as a single position argument.
```
Comment (bowlofeggs) [inline docs/consuming.rst]:
```suggestion
object as a single positional argument.
```

Comment (jeremycline) [reply docs/consuming.rst]: Fixed, thanks!

**Comment by jeremycline** [comment]
> I think it would have helped me today if we add an example of calling that example class via the CLI. i.e. fedora-messaging --conf /work/my_config.toml consume --callback=modname:SaveMessage

I've added a note following the example class.

**Comment by dustymabe** [comment]
> > I think it would have helped me today if we add an example of calling that example class via the CLI. i.e. fedora-messaging --conf /work/my_config.toml consume --callback=modname:SaveMessage
> 
> I've added a note following the example class.

👍

**Pull request merged**